config/checkstyle-resources-suppressions.xml

```diff
@@ -705,5 +705,7 @@
     files="[\\/]test[\\/]resources[\\/]com[\\/]puppycrawl[\\/]tools[\\/]checkstyle[\\/]grammar[\\/]java14[\\/]InputJava14Records\.java"/>
   <suppress checks="FileLength"
     files="[\\/]test[\\/]resources-noncompilable[\\/]com[\\/]puppycrawl[\\/]tools[\\/]checkstyle[\\/]grammar[\\/]antlr4[\\/]InputAntlr4AstRegressionUncommon\.java"/>
+  <suppress checks="FileLength"
+    files="[\\/]test[\\/]resources-noncompilable[\\/]com[\\/]puppycrawl[\\/]tools[\\/]checkstyle[\\/]checks[\\/]coding[\\/]unusedtryresourceshouldbeunnamed[\\/]InputUnusedTryResourceShouldBeUnnamedNested\.java"/>
```

Comment:
Please split input into two files, we highly encourage contributors to not make huge input, it complicated debug